### ISSUE TITLE
How to edit a previously created page?

### ISSUE THREAD

**Issue by JakeBV** [mercuree/html-telegraph-poster#5]
I have a page that I created earlier. How can I change it? Something like code:
`link_page = 'https://telegra.ph/mypagetoedit'`
`t.edit(link_page, text=link_page.text + 'my new text')`

**Comment by mercuree** [comment]
Ok, i will fix it soon.
Workaround:
```python
t =  TelegraphPoster(access_token='<your access token>')
t.author_url = ''
t.path = 'https://telegra.ph/mypagetoedit'
t.edit_page('title', 'author', '<p>html here</p>')
```

**Comment by mercuree** [comment]
Fixed 8d4a429ffd343099286ba115a8ace765d3cda063
edit_page now accepts path parameter
```python
t.edit_page('title', 'author', '<p>html here</p>', path='/page-path-05-06')
```
